### ISSUE TITLE
Not enough love for firefox

### ISSUE THREAD

**Issue by vessd** [Mathspy/binary-clock#1]
Heart-shaped clocks are incorrectly displayed on firefox 60.0.2, 60.5.2 (ESR) and Andriod 60.0.
![Снимок экрана_2019-03-15_21-06-05](https://user-images.githubusercontent.com/8783865/54452547-7bc11b00-4766-11e9-9651-395530cab0a0.png)

**Comment by Mathspy** [comment]
Damn... XD Any idea why? Firefox always hated my SVGs for some reason

**Comment by vessd** [comment]
I have no ideas, I'm not good at web programming.
Google knows a lot of problems with Firefox and SVG. This could be a bug in Firefox.

**Comment by Mathspy** [comment]
Yeah I sadly suspect that as it's not my first time stumbling upon weird inconsistencies with Firefox's SVG implementation (it works on Safari and Chrome if anything)
But I will probably still look into it later, maybe it's not too hard to fix! :3

Thanks for the catch! :heart:
